WorldCat/worldcat.js

```diff
@@ -55,9 +57,9 @@ module.exports = function (returnToMaster) {
 
                 });
 
-                linkCounter++;
-
-                if (linkCounter > total_links) {
+                // worldcat only shows the first 5000 pages
+                console.log(linkCounter)
+                if (linkCounter == 500) {
```

Comment:
Nope, that's right as we get 10 entries per link, so thats 500*10=5000 in total. 
But I just removed an unnecessary comment from the cli.js that is no longer needed (because earlier the script ran out of RAM )